### PR TITLE
okta platinum sponser removed

### LINKED ISSUE #7223: README: remove Okta Platinum Sponsor

### PR DESCRIPTION
**Pull request by sagar-subedi**
Fixes #7223: Removes Okta from platinum section of sponsors in Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,11 +287,6 @@ Support this project by becoming a sponsor! [Become a sponsor](https://opencolle
 <table>
   <tbody>
     <tr>
-      <td align="center" valign="middle">
-        <a href="https://dev.entando.org/jhipster" target="_blank">
-          <img width="200em" src="https://www.jhipster.tech/images/open-collective/entandoe.png">
-        </a>
-      </td>
```
Comment (pascalgrimaud) [inline README.md:294]: wrong sponsor, it's okta, not entando

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]
you can completely delete the `### Platinum Sponsors` section, including table of Okta

**Comment by sagar-subedi** [comment]
okta along with the who platinum sponsor removed, and entando is left untouched.

**Comment by sagar-subedi** [comment]
*whole

File: README.md [hunk 2]
```diff
@@ -268,26 +268,13 @@ Support this project by becoming a sponsor! [Become a sponsor](https://opencolle
...
-  </tbody>
-</table>
 
 ### Gold Sponsors
 
 <table>
   <tbody>
     <tr>
-      <td align="center" valign="middle">
+     <td align="center" valign="middle">
```
Comment (pascalgrimaud) [inline README.md:277]:
```suggestion
      <td align="center" valign="middle">
```

File: README.md [hunk 3]
```diff
@@ -268,19 +268,6 @@ Support this project by becoming a sponsor! [Become a sponsor](https://opencolle
...
-      <td align="center" valign="middle">
-        <a href="https://developer.okta.com/signup?utm_source=JHipster&utm_medium=logo&utm_campaign=Platinum-Sponsor" target="_blank">
-          <img width="425em" src="https://www.jhipster.tech/images/open-collective/okta.png">
-        </a>
-      </td>
-    </tr>
-  </tbody>
-</table>
 
 ### Gold Sponsors
```
Comment (pascalgrimaud) [inline README.md:273]:
```suggestion
### Gold Sponsors

```

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

**Review by pascalgrimaud — APPROVED** [review]

**Comment by pascalgrimaud** [comment]
Thanks and congrats for your 1st contribution @sagar-subedi

**Comment by sagar-subedi** [comment]
Feels like a great beginning to my open source journey. Thanks to the community.

**Pull request merged**